syncapi/storage/postgres/receipt_table.go

```diff
@@ -96,7 +96,7 @@ func (r *receiptStatements) UpsertReceipt(ctx context.Context, txn *sql.Tx, room
 }
 
 func (r *receiptStatements) SelectRoomReceiptsAfter(ctx context.Context, roomIDs []string, streamPos types.StreamPosition) (types.StreamPosition, []api.OutputReceiptEvent, error) {
-	lastPos := streamPos
```

Comment:
By doing this, if there are no receipts, we will return `0`. Is this intentional?

Reply:
Yes, because the call site for `RoomReceiptsAfter` specifically checks for that case.